data/json/construction.json

```diff
@@ -461,7 +461,7 @@
             [ [ "glass_sheet", 1 ] ]
         ],
         "pre_terrain" : "t_window_empty",
-        "post_terrain" : "t_window"
+        "post_terrain" : "t_window_no_curtains"
     },{
         "type" : "construction",
         "description" : "Build Window", "//": "Step 3: window with curtains",
```

Comment:
Not sure if this one should even stay, you won't be able to build t_window just find them in stores.